crates/solvers/src/infra/config/mod.rs

```diff
@@ -3,6 +3,16 @@ use std::fmt::Debug;
 pub mod baseline;
 pub mod dex;
 pub mod legacy;
+pub mod naive;
+
+/// Parameters used to calculate the revert risk of a solution.
+#[derive(Debug, Clone)]
+pub struct RiskParameters {
```

Comment:
Is this used anywhere (it looks like you are directly using the domain model in the other files)? Feels like it should go into its own file, shouldn't it?

Reply:
`slippage::Limits` is also defined in `domain` but used in `infra` where the config is loaded. I think it's expected from `infra` to import `domain` objects.

Reply:
Btw, you are right, this structure is not used anywhere, I will delete it.